sig/anyway_config.rbs

```diff
@@ -67,8 +68,8 @@ module Anyway
     def self.attr_config: (*Symbol args, **untyped) -> void
     def self.defaults: -> Hash[String, untyped]
     def self.config_attributes: -> Array[Symbol]?
-    def self.required: (*Symbol names) -> void
-    def self.required_attributes: -> Array[Symbol]
+    def self.required: (*Symbol names, { env: String | Symbol | Array[String | Symbol] | { except: String | Symbol | Array[String | Symbol] } }?) -> void
+    def self.required_attributes: -> Array[Symbol | ({ env: Symbol | String | Array[String | Symbol] | { except: String | Symbol | Array[String | Symbol] } })?]
```

Comment:
As I can see, we put environment-dependent names into a different variable; can we have `{env: ... }` here?

(I think, we should put everything into the existing var, `required_attributes`; and for backward compatibility the current signature (either a symbol, or a hash) makes sense)

Reply:
What do you mean? Make env untyped like `{ env: untyped | { except: untyped } }`?